### PR TITLE
updated table shema and completed all queries

### PR DESCRIPTION
**Pull request by AyoMoses1**
I updated the table schema by adding a species column and completed all queries. Kindly find an attachment of the screenshots of all requirements
Screenshot (334)
Screenshot (335)
Screenshot (336)
Screenshot (337)
Screenshot (338)
Screenshot (339)
Screenshot (340)
Screenshot (341)
Screenshot (342)
Screenshot (343)
Screenshot (344)
Screenshot (345)
Screenshot (346)
Screenshot (347)
Screenshot (348)
Screenshot (349)
Screenshot (332)
Screenshot (333)

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -3,6 +3,7 @@
 CREATE TABLE animals(
   id                INT GENERATED ALWAYS AS IDENTITY,
   name              VARCHAR(100),
+  species           VARCHAR(255),
```
Comment (franclobo) [inline schema.sql]: - [ ] You have added a new column directly in the table, but for this practice please 🙏🏽 use the psql terminal to add the new column 👇🏽 

`ALTER TABLE animals ADD species varchar(255);`

File: queries.sql [hunk 2]
```diff
@@ -5,4 +5,36 @@ SELECT date_of_birth from animals where name = 'Agumon' or name = 'Pikachu'
...
-SELECT * from animals where weight_kg >= 10.4 and weight_kg <= 17.3
\ No newline at end of file
+SELECT * from animals where weight_kg >= 10.4 and weight_kg <= 17.3
+
+
+BEGIN;
+UPDATE animals
+SET species = 'unspecified'
+ROLLBACK;
+
```
Comment (franclobo) [inline queries.sql:15]: - [ ] Kindly verify the change was undone using the following line at the end 👇🏽 

`SELECT species from animals;`

File: queries.sql [hunk 3]
```diff
@@ -5,4 +5,36 @@ SELECT date_of_birth from animals where name = 'Agumon' or name = 'Pikachu'
...
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT;
+
```
Comment (franclobo) [inline queries.sql]: - [ ] Please verify that the change persists after commit 👇🏽 

`SELECT species from animals;`

File: queries.sql [hunk 4]
```diff
@@ -5,4 +5,36 @@ SELECT date_of_birth from animals where name = 'Agumon' or name = 'Pikachu'
...
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
```
Comment (franclobo) [inline queries.sql:28]: - [ ] Kindly verify the change was undone 👍🏽

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @AyoMoses1, 

# ♻️ Required Changes ♻️

so-close-jim-halpert

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
 
- Correct data ✔️
- Correct Git flow ✔️ 
- Descriptive PR ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Please 🙏🏽 order the screenshots on your PR for professional documentation, it is difficult to review if they aren't ordered.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShadyShawkat — APPROVED** [review]
## Hi @AyoMoses1, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**